src/table/components/virtualized-table/hooks/use-totals.ts

```diff
@@ -1,24 +1,12 @@
 import { useMemo } from 'react';
 import { getTotalPosition } from '../../../../handle-data';
 import { TableLayout } from '../../../../types';
-import { HEADER_HEIGHT, HEADER_AND_TOTALS_HEIGHT } from '../constants';
 
 export interface Totals {
   atBottom: boolean;
   atTop: boolean;
-  shrinkBodyHeightBy: number;
 }
 
-const useTotals = (layout: TableLayout) => {
-  const totalsPosition = getTotalPosition(layout);
-
-  return useMemo(
-    () => ({
-      ...totalsPosition,
-      shrinkBodyHeightBy: totalsPosition.atTop || totalsPosition.atBottom ? HEADER_AND_TOTALS_HEIGHT : HEADER_HEIGHT,
-    }),
-    [totalsPosition]
-  );
-};
+const useTotals = (layout: TableLayout) => useMemo(() => getTotalPosition(layout), [layout]);
```

Comment:
do you still need this hook? It it would be a one-liner where the hook is used instead

Reply:
Agree. Will remove hook 👍